### PR TITLE
Add total numbers of signatures inside or above the bars

### PR DESCRIPTION
**Pull request by Rekyt**

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -119,8 +124,12 @@ walk2(
 d %>%
   ggplot(aes(y = fct_rev(Candidat), fill = fct_rev(`Date de publication`))) +
     geom_histogram(stat = "count") +
-    scale_fill_brewer(palette = "Greens", direction = -1) +
+    scale_fill_brewer(palette = "Greens", direction = 1) +
```
Comment (Bisaloo) [inline index.Rmd]: Is this change on purpose?

Comment (Rekyt) [reply index.Rmd]: Woops, absolutely not. It appeared different on my laptop compared to the website.

Comment (Bisaloo) [reply index.Rmd]:
```suggestion
    scale_fill_brewer(palette = "Greens", direction = -1) +
```

**Comment by Bisaloo** [comment]
Thanks!
